### ISSUE TITLE
ACSF scaler seems to give out range values for some scaler types

### ISSUE THREAD

**Issue by hghcomphys** [hghcomphys/jaxip#40]
For example, by setting `scale_min/max`=(-1, 1) and `scaler_type`=scaler_center, scaled descriptor values are no longer zero-mean. This can make difficult for optimizer to train the model efficiently. 

As an idea,  `batch normalization` can improve the model training.
